tests/test_create_gatorgradle_yml.py

```diff
@@ -0,0 +1,44 @@
+"""This module tests the configator module."""
+
+import os
+
+from configator import create_gatorgradle_yml
+
+
+def test_create_gatorgrader_makes_config_directory(mocker):
+    """tests create_gatorgrader function, specifically
+    the creation of the config file.
+    """
+    mock_open = mocker.mock_open()
+    mocker.patch('builtins.open', mock_open)
+    mocker.patch('os.mkdir')
+    create_gatorgradle_yml.create_gatorgrader()
+    os.mkdir.assert_called_once_with('config')
```

Comment:
You mock `open` here as well as `os.mkdir` -- you should also assert that `open` is called with the correct filename.